### ISSUE TITLE
Nodegroup `update_default_version` config should be parameterized

### ISSUE THREAD

**Issue by jj-mino** [aws-ia/terraform-aws-eks-blueprints#1589]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

#### What is the outcome that you are trying to reach?

When `create_launch_template=true` is set at the nodegroups, they are 'upgraded' every time that the launch_template is updated.
I would like to be able to set the [update_default_version](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/modules/aws-eks-managed-node-groups/managed-launch-templates.tf#L6) so we could update the launch template without need to force nodegroups updates, so we could have more control on 'when' do we want to perform that.

#### Describe the solution you would like
`update_default_version` should be set by a TF var

#### Describe alternatives you have considered

Couldn't find any workaround to above approach

#### Additional context
```

# module.eks_blueprints.module.aws_eks_managed_node_groups["<nodegroup>"].aws_launch_template.managed_node_groups[0] will be updated in-place
~ resource "aws_launch_template" "managed_node_groups" {
    ~ default_version         = 3 -> (known after apply)  # I would expect this to be skipped so nodegroup below is not updated
      id                      = "lt-<id>"
    ~ image_id                = "ami-0d9781fbca2991de5" -> "ami-0e1597f2056965cab"
    ~ latest_version          = 3 -> (known after apply)
      name                    = "<myawasomelaunchtemplate>"
      tags                    = {}
      # (10 unchanged attributes hidden)
      # (7 unchanged blocks hidden)
  }

# module.eks_blueprints.module.aws_eks_managed_node_groups["<nodegroup>"].aws_eks_node_group.managed_ng will be updated in-place
~ resource "aws_eks_node_group" "managed_ng" {
      tags                   = {
          "Name"                                        = "<nodegroup>"
          "k8s.io/cluster-autoscaler/<clustername>" = "owned"
          "k8s.io/cluster-autoscaler/enabled"           = "TRUE"
          "kubernetes.io/cluster/<clustername>"     = "owned"
          "managed-by"                                  = "terraform-aws-eks-blueprints"
      }
      # (17 unchanged attributes hidden)
    ~ launch_template {
          id      = "lt-<id>"
          name    = "<myawasomelaunchtemplate>"
        ~ version = "3" -> (known after apply)
      }
      # (4 unchanged blocks hidden)
  }
```

**Comment by jj-mino** [comment]
I can create a PR for it if you think the change makes sense

**Comment by bryantbiggs** [comment]
closed in #1590
